### PR TITLE
k6-summary: update to v0.1.0

### PR DESCRIPTION
**Pull request by olegbespalov**
## Please fill in this template.

- [x] Use a meaningful title for the Pull Request. Include the name of the jslib added/modified.
- [x] Fill the description section of the Pull Request. 
- [x] Test the change in your code, and ensure the `npm run test` command succeeds.
- [x] Run `yarn run generate-homepage` locally and verify the new homepage `/lib/index.html` file looks legit.

**Select one of these and delete the others**:

*If adding a new jslib*:
- [ ] The Pull Request creates a `/lib/{jslib_name}` folder.
- [ ] The Pull Request creates a `/lib/{jslib_name}/{desired_version}` folder.
- [ ] The `/lib/{jslib_name}/{desired_version}/index.js` file containing the jslib's code bundle exists.
- [ ] The Pull Request updates the `supported.json` file to contain an entry for the newly added jslib and its `{desired_version}`, as in the following example:
```JSON
{
  "{jslib_name}": {
    // Available package versions
    "versions": [
      "{desired_version}"
    ],

    // (optional) Documentation's or repository's URL
    "docs-url": "{documentation_or_repository_url}",

    // (optional) As a default, the homepage will point to
    // a package's bundle's index.js. If your package's main
    // bundle name is different; set it here (see the AWS
    // package for instance).
    "bundle-filename": "{index.js}"
}
```
- [ ] Tests have been added to `/tests/basic.js` and `/tests/testSuite.js` to ensure that the added jslib is importable and runnable by k6.

*If publishing a new version of an existing jslib*:
- [x] The Pull Request is labeled with the `version bump` label.
- [x] The Pull Request adds a `/lib/{jslib_name}/{desired_version}` folder.
- [x] The Pull Request adds a `/lib/{jslib_name}/{desired_version}/index.js` file containing the jslib's code bundle.
- [x] The Pull Request updates the `supported.json` file to contain an entry for the newly added jslib version, as in the following example:
```JSON
{
  "my-lib": [
    "1.0.2",
    // Use semantic versioning
    "{desired_version}"
  ]
}
```
- [x] The Pull Request adds the relevant tests to the `/tests/basic.js` and `/tests/testSuite.js` files to ensure that the new version of the jslib is importable and runnable by k6.
- Merge the Pull Request once it is green. PRs adding new jslib versions do not require to get a review to be merged :rocket:.

### CODE REVIEW

**Comment by olegbespalov** [comment]
The build was produced by the CI and attached as release artifact https://github.com/grafana/k6-jslib-summary/releases/tag/v0.1.0

File: lib/k6-summary/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# k6-summary
+
+This functions that can be used to generate different end-of-test summary reports when used in the `handleSummary()` callback: [https://k6.io/docs/results-visualization/end-of-test-summary](https://k6.io/docs/results-visualization/end-of-test-summary/).
```
Comment (going-confetti) [inline lib/k6-summary/README.md:3]: I think we should link to https://grafana.com/docs/k6/latest/results-output/end-of-test/, unless there's a separate issue for updating links in this repo

File: lib/k6-summary/0.1.0/index.js
Comment (going-confetti) [inline lib/k6-summary/0.1.0/index.js:1]: I'm not too familiar with the codebase and I've noticed that some libs/versions are minified and some aren't, so I wanted to ask: do we have guidelines for this?

Comment (olegbespalov) [reply lib/k6-summary/0.1.0/index.js:1]: yep, that was the case and created some kind of confusion, but we'd like to try set up most of the projects with minification (like in a PR https://github.com/grafana/k6-jslib-summary/pull/8) and at some point automate updating of this repo code, by using the artifacts from the releases

File: lib/k6-summary/README.md (hunk 1 above)
Comment (olegbespalov) [reply lib/k6-summary/README.md:3]: Yeah, it's a good point :+1:

Comment (olegbespalov) [reply lib/k6-summary/README.md:3]:
```suggestion
This functions that can be used to generate different end-of-test summary reports when used in the `handleSummary()` callback: [https://grafana.com/docs/k6/latest/results-output/end-of-test/](https://grafana.com/docs/k6/latest/results-output/end-of-test/).
```

**Review by going-confetti — APPROVED** [review]
LGTM!

File: lib/k6-summary/0.1.0/index.js
Comment (mstoykov) [reply lib/k6-summary/0.1.0/index.js:1]: IMO minifying *with* adding the map file doesn't make much sense for k6.

k6 does by default download and load map files as to make developer experience better. As such all the savings from not minifying will be not existant.

I am pretty sure we minify others for:
1. k6 did not have source map support
2. this is what you just do with js ... :shrug: 

I guess we can make k6 stop doing this by default, but then the UX will be worse whenever there is an exception.

Comment (olegbespalov) [reply lib/k6-summary/0.1.0/index.js:1]: Okay, than I had a wrong assumption. How about just minify the scripts then?

**Review by mstoykov — APPROVED** [review]

File: lib/k6-summary/0.1.0/index.js
Comment (mstoykov) [reply lib/k6-summary/0.1.0/index.js:1]: IMO - it doesn't matter as much either way - which is why I approved this.

In the end on each test *run* k6 will download those files *once*.

If a user use `k6 archive`and then runs the archive - only on the `archive` command they will be downloaded.

Which also means that in the cloud only on `k6 cloud` but ont on just rerunnign from the UI or if it is scheduled.

All in all I expect the amount of downloads is not significant to matter whether:
1. we minify it.
2. k6 has to download a map as well as 1
3. we do not modify it k6 has to download slightly bigger file.

Comment (codebien) [reply lib/k6-summary/0.1.0/index.js:1]: Then to go back to the original question, it seems to me that the guideline we would like to have is to _not_ minify the scripts. And, the reason is so we can keep their projects simpler. Am I getting it right?

**Review by codebien — APPROVED** [review]

File: lib/k6-summary/0.1.0/index.js
Comment (olegbespalov) [reply lib/k6-summary/0.1.0/index.js:1]: @codebien I believe that you don't. The main point of the discussion right now are source maps. Minifying and source maps are related, but the minifying doesn't require source maps.

**Pull request merged**